src/test/java/org/kohsuke/file_leak_detector/instrumented/FileDemo.java

```diff
@@ -52,6 +58,12 @@ public void openSocket(Object obj) {
 
         @Override
         public void close(Object obj) {
+			// sometimes java.util.zip.ZipFile$CleanableResource$FinalizableResource.finalize()
+			// will kick in and will close a ZipFile, thus we ignore the corresponding objects here
+			if (obj.getClass().getSimpleName().contains("URLJarFile")) {
+				return;
+			}
+
```

Comment:
Thanks very much!.  Optional indentation change to make it consistent with the indentation immediately below it.
```suggestion
           // sometimes java.util.zip.ZipFile$CleanableResource$FinalizableResource.finalize()
           // will kick in and will close a ZipFile, thus we ignore the corresponding objects here
           if (obj.getClass().getSimpleName().contains("URLJarFile")) {
               return;
           }

```